src/Symfony/Component/Routing/Tests/RouteCompilerTest.php

```diff
@@ -47,7 +47,7 @@ public function provideCompileData()
                 'Route with a variable',
                 ['/foo/{bar}'],
                 '/foo', '#^/foo/(?P<bar>[^/]++)$#sD', ['bar'], [
-                    ['variable', '/', '[^/]++', 'bar'],
+                    ['variable', '/', '[^/]++', 'bar', false, false],
```

Comment:
It would be great to not have to change these lines - that would also allow preserving the serialization format below.
I'm not sure it applies here, but changing the serialized payload should be avoided when possible, to be sure we don't break existing serialized routes. Doable?

Reply:
@nicolas-grekas , I think, yes. I will work on it soon

Reply:
@nicolas-grekas , it will not break existing serialized routes because of this line: https://github.com/symfony/symfony/pull/29770/files#diff-b1846de60f5a42e8aef1e3aa69649368R160. In this PR I didn't change the structure, just used one more array's index (5) to keep `(bool) $important`. And because of this I have to make sure the previous flag (4) `(bool) $useUtf8` is always set.

I can try to add a test for this.